configure

```diff
@@ -523,43 +526,45 @@ if (!exists $platforminfo{$opts{'target'}}->{'needs_i2jrt_corba'}) {
 
 ## compiler
 
-if ($opts{'compiler'}) {
-  my $standard = 0;
-  for my $stdcomp (@{$platforminfo{$opts{'host'}}->{'compilers'}}) {
-    $standard = 1 if $opts{'compiler'} eq $stdcomp;
-  }
-  $opts{'nonstdcompiler'} = 1 unless $standard;
-}
-else {
-  print "Auto-detecting compiler\n" if $opts{'verbose'};
-  for my $stdcomp (@{$platforminfo{$opts{'host'}}->{'compilers'}}) {
-    my $path = which($stdcomp);
-    if ($path) {
-      print "Found $stdcomp at: $path\n" if $opts{'verbose'};
-      $opts{'compiler'} = $stdcomp;
-      last;
+if ($has_host_compiler) {
+  if ($opts{'compiler'}) {
+    my $standard = 0;
+    for my $stdcomp (@{$platforminfo{$opts{'host'}}->{'compilers'}}) {
+      $standard = 1 if $opts{'compiler'} eq $stdcomp;
     }
+    $opts{'nonstdcompiler'} = 1 unless $standard;
   }
-  if (!defined $opts{'compiler'}) {
-    die "ERROR: Can't find a compiler, set PATH or run this script with the ".
-      "--compiler option.\n" . ($is_windows ? "  For Microsoft Visual C++, ".
-                                "run this script from the Visual Studio ".
-                                "Command Prompt.\n" : '') . "Stopped";
+  else {
+    print "Auto-detecting compiler\n" if $opts{'verbose'};
+    for my $stdcomp (@{$platforminfo{$opts{'host'}}->{'compilers'}}) {
+      my $path = which($stdcomp);
+      print "$stdcomp: $path\n";
```

Comment:
Verbose debug only?

Reply:
It was a debugging statement I added, I'll remove it.